docs/buildah-add.md

```diff
@@ -23,6 +23,10 @@ Defaults to false.
 Note: You can also override the default value of --add-history by setting the
 BUILDAH\_HISTORY environment variable. `export BUILDAH_HISTORY=true`
 
+**--chmod** *permissions*
+
+Sets the access permissions of the destination content.
```

Comment:
Can we specify the input format?  A `--chmod +x`, for instance, won't work.

Reply:
done